src/buildCommon.js

```diff
@@ -169,6 +169,20 @@ const makeOrd = function(group, options, type) {
 };
 
 /**
+ * Combine as many characters as possible in the given array of characters
+ * via their tryCombine method.
+ */
+const tryCombineChars = function(chars) {
```

Comment:
Maybe this should be:
```
const tryCombineTextNodes = function(nodes) {
```
It's possible that we might also get passed a `documentFragment` so we should probably add a `tryCombine` method on that class as well just in case.